### ISSUE TITLE
Redigo version change in go.mod

### ISSUE THREAD

**Issue by zwass** [throttled/throttled#86]
Due to some confusion during the migration of the Redigo project (see https://github.com/gomodule/redigo/issues/366), some projects (see https://github.com/RediSearch/redisearch-go/issues/98) ended up with the orphaned (and older) 2.0.0+incompatible version in the go.mod. This seems to have happened to Throttled.

When I "upgrade" Redigo in https://github.com/fleetdm/fleet with `go get -u github.com/gomodule/redigo@v1.8.4`, the Go tooling downgrades Throttled to 2.2.5 (the last version before the `go.mod` file was added).

Can we update the `go.mod` to include Redigo 1.8.4 (the current latest version)? Doing this and cutting a release would allow our project to use the current version of Throttled along with the current version of Redigo, without needing a manual `replace` directive in the `go.mod`.

Thanks!

**Comment by zwass** [comment]
My stopgap solution is the following in `go.mod`:

```
replace github.com/gomodule/redigo => github.com/gomodule/redigo v1.8.4
```

It doesn't seem possible to instead use the `replace` on `github.com/throttled/throttled`, as the compiler then complains about `github.com/gomodule/redigo@2.0.0+incompatible` being missing.

**Comment by brandur** [comment]
Released version 2.7.2 with Redigo 1.8.4. Hopefully that'll help!

**Comment by zwass** [comment]
Thank you!
